kitchen.yml

```diff
@@ -99,17 +100,26 @@ suites:
     run_list:
       - recipe[nftables-test]
 
+  - name: firewalldnew
+    includes:
+      - debian-11
+      - fedora-34
+    run_list:
+      - recipe[firewalld-test]
+
   - name: windows
     excludes:
       - almalinux-8
       - amazonlinux-2
       - debian-9
       - debian-10
+      - debian-11
       - centos-7
       - centos-8
+      - fedora-34
+      - fedora-latest
       - freebsd-11
       - freebsd-12
-      - fedora-latest
```

Comment:
this should still be here since this isn't a windows platform and fedora-latest is still in your list of platforms.

Reply:
Sorry for the mess. This wasn't actually gone. I just half-heartedly sorted the list alphabetically. Anyway it is sorted now, and the fedora-latest should still be present.